<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="10" passed="10">
  <reporter-output>
  </reporter-output>
  <suite name="com.mid_testing_project.services.VisitationServiceNGTest" duration-ms="5650" started-at="2021-04-05T21:51:28 CAT" finished-at="2021-04-05T21:51:34 CAT">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="5650" started-at="2021-04-05T21:51:28 CAT" finished-at="2021-04-05T21:51:34 CAT">
      <class name="com.mid_testing_project.services.VisitationServiceNGTest">
        <test-method status="PASS" signature="initializeDatabase()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="initializeDatabase" is-config="true" duration-ms="3650" started-at="2021-04-05T21:51:28 CAT" finished-at="2021-04-05T21:51:32 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDatabase -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="106" started-at="2021-04-05T21:51:32 CAT" finished-at="2021-04-05T21:51:32 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testFindAllVisitations()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testFindAllVisitations" duration-ms="337" started-at="2021-04-05T21:51:32 CAT" finished-at="2021-04-05T21:51:32 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFindAllVisitations -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="112" started-at="2021-04-05T21:51:32 CAT" finished-at="2021-04-05T21:51:32 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="88" started-at="2021-04-05T21:51:32 CAT" finished-at="2021-04-05T21:51:32 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testFindAllVisitationsInvalidPrison()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testFindAllVisitationsInvalidPrison" duration-ms="3" started-at="2021-04-05T21:51:32 CAT" finished-at="2021-04-05T21:51:32 CAT">
          <exception class="com.mid_testing_project.exceptions.InvalidPrisonException">
            <message>
              <![CDATA[prison does not exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.mid_testing_project.exceptions.InvalidPrisonException: prison does not exist
at com.mid_testing_project.services.VisitationService.findAllVisitations(VisitationService.java:76)
at com.mid_testing_project.services.VisitationServiceNGTest.testFindAllVisitationsInvalidPrison(VisitationServiceNGTest.java:87)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- com.mid_testing_project.exceptions.InvalidPrisonException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFindAllVisitationsInvalidPrison -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="146" started-at="2021-04-05T21:51:32 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="99" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testFindAllVisitationsOnCertainDate()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testFindAllVisitationsOnCertainDate" duration-ms="36" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFindAllVisitationsOnCertainDate -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="75" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="77" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testFindAllVisitationsOnCertainDateInvalidDate()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testFindAllVisitationsOnCertainDateInvalidDate" duration-ms="6" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <exception class="com.mid_testing_project.exceptions.DateNotYetOccurredException">
            <message>
              <![CDATA[date not yet occurred]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.mid_testing_project.exceptions.DateNotYetOccurredException: date not yet occurred
at com.mid_testing_project.services.VisitationService.findAllVisitationsOnCertainDate(VisitationService.java:86)
at com.mid_testing_project.services.VisitationServiceNGTest.testFindAllVisitationsOnCertainDateInvalidDate(VisitationServiceNGTest.java:103)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- com.mid_testing_project.exceptions.DateNotYetOccurredException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFindAllVisitationsOnCertainDateInvalidDate -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="72" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="59" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testFindAllVisitationsOnCertainDateInvalidPrison()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testFindAllVisitationsOnCertainDateInvalidPrison" duration-ms="4" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <exception class="com.mid_testing_project.exceptions.InvalidPrisonException">
            <message>
              <![CDATA[prison does not exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.mid_testing_project.exceptions.InvalidPrisonException: prison does not exist
at com.mid_testing_project.services.VisitationService.findAllVisitationsOnCertainDate(VisitationService.java:83)
at com.mid_testing_project.services.VisitationServiceNGTest.testFindAllVisitationsOnCertainDateInvalidPrison(VisitationServiceNGTest.java:98)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- com.mid_testing_project.exceptions.InvalidPrisonException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFindAllVisitationsOnCertainDateInvalidPrison -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="80" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="89" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testVisit()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testVisit" duration-ms="77" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testVisit -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="45" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="55" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testVisitInvalidPrisoner()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testVisitInvalidPrisoner" duration-ms="7" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <exception class="com.mid_testing_project.exceptions.InvalidPrisonerException">
            <message>
              <![CDATA[prisoner does not exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.mid_testing_project.exceptions.InvalidPrisonerException: prisoner does not exist
at com.mid_testing_project.services.VisitationService.visit(VisitationService.java:49)
at com.mid_testing_project.services.VisitationServiceNGTest.testVisitInvalidPrisoner(VisitationServiceNGTest.java:60)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- com.mid_testing_project.exceptions.InvalidPrisonerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testVisitInvalidPrisoner -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="48" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:33 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="55" started-at="2021-04-05T21:51:33 CAT" finished-at="2021-04-05T21:51:34 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testVisitInvalidTime()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testVisitInvalidTime" duration-ms="10" started-at="2021-04-05T21:51:34 CAT" finished-at="2021-04-05T21:51:34 CAT">
          <exception class="com.mid_testing_project.exceptions.InvalidVisitationTimeException">
            <message>
              <![CDATA[time does not exist]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.mid_testing_project.exceptions.InvalidVisitationTimeException: time does not exist
at com.mid_testing_project.services.VisitationService.visit(VisitationService.java:52)
at com.mid_testing_project.services.VisitationServiceNGTest.testVisitInvalidTime(VisitationServiceNGTest.java:66)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- com.mid_testing_project.exceptions.InvalidVisitationTimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testVisitInvalidTime -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="74" started-at="2021-04-05T21:51:34 CAT" finished-at="2021-04-05T21:51:34 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="43" started-at="2021-04-05T21:51:34 CAT" finished-at="2021-04-05T21:51:34 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testVisitNotVisitationDay()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testVisitNotVisitationDay" duration-ms="8" started-at="2021-04-05T21:51:34 CAT" finished-at="2021-04-05T21:51:34 CAT">
          <exception class="com.mid_testing_project.exceptions.InvalidVisitationDateException">
            <message>
              <![CDATA[date is not enabled]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.mid_testing_project.exceptions.InvalidVisitationDateException: date is not enabled
at com.mid_testing_project.services.VisitationService.visit(VisitationService.java:59)
at com.mid_testing_project.services.VisitationServiceNGTest.testVisitNotVisitationDay(VisitationServiceNGTest.java:76)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- com.mid_testing_project.exceptions.InvalidVisitationDateException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testVisitNotVisitationDay -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="37" started-at="2021-04-05T21:51:34 CAT" finished-at="2021-04-05T21:51:34 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
        <test-method status="PASS" signature="populateTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="populateTables" is-config="true" duration-ms="44" started-at="2021-04-05T21:51:34 CAT" finished-at="2021-04-05T21:51:34 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- populateTables -->
        <test-method status="PASS" signature="testVisitOutOfBoundsVisitation()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="testVisitOutOfBoundsVisitation" duration-ms="6" started-at="2021-04-05T21:51:34 CAT" finished-at="2021-04-05T21:51:34 CAT">
          <exception class="com.mid_testing_project.exceptions.OutOfBoundsVisitationTimeException">
            <message>
              <![CDATA[time is out of boundaries]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.mid_testing_project.exceptions.OutOfBoundsVisitationTimeException: time is out of boundaries
at com.mid_testing_project.services.VisitationService.visit(VisitationService.java:62)
at com.mid_testing_project.services.VisitationServiceNGTest.testVisitOutOfBoundsVisitation(VisitationServiceNGTest.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- com.mid_testing_project.exceptions.OutOfBoundsVisitationTimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testVisitOutOfBoundsVisitation -->
        <test-method status="PASS" signature="cleanTables()[pri:0, instance:com.mid_testing_project.services.VisitationServiceNGTest@1ed4004b]" name="cleanTables" is-config="true" duration-ms="49" started-at="2021-04-05T21:51:34 CAT" finished-at="2021-04-05T21:51:34 CAT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanTables -->
      </class> <!-- com.mid_testing_project.services.VisitationServiceNGTest -->
    </test> <!-- Command line test -->
  </suite> <!-- com.mid_testing_project.services.VisitationServiceNGTest -->
</testng-results>
